Makefile

```
@@ -109,11 +54,14 @@ endif
 
 .PHONY: all modules clean help show check always-update install-models
 
-clean: # add more prerequisites for clean below
-	$(RM) -r $(SUB_VENV)
+clean: network-clean images-clean
```

```suggestion
clean: network-clean
```
I'd prefer to make image cleaning explicit, so that it is not necessary to download again when "accidentally" called clean. Help output also would have to be adapted accordingly

---

Yes, that's to be considered...